autotest/test_gwf_disu01.py

```diff
@@ -131,7 +134,7 @@ def test_disu_idomain_simple():
     budobj = flopy.utils.CellBudgetFile(fname, precision="double")
     flowja = budobj.get_data(text="FLOW-JA-FACE")[0].flatten()
     assert flowja.shape[0] == 126
-
+    shutil.rmtree(ws, ignore_errors=True)
```

Comment:
Should we think about checking a --keep argument before removing?

Reply:
Added a general function to look for the `--keep` argument in `framework.py` and added check in `autotest/test_gwf_disu01.py`